.nvmrc

```
@@ -1 +1 @@
-v16.16.0
+v18.17.1
```

```suggestion
v18.17.1
```
It might be worth seeing if node 20 works. Node 18 has been relatively unstable with memory leaks, bugs, and regressions in minor releases

---

I have a preference for LTS in production but if you feel like v20 is preferable then I'm open to it

---

It's pretty close. Might be best to use LTS and then consider upgrading if we happen to run into problems.